src/App.js

```diff
@@ -1,199 +1,178 @@
-import React, { Component } from 'react';
+import React, { Component, useState, useEffect } from 'react';
 import uuid from 'uuid';
 import './App.css';
 import GameItem from './components/GameItem.js';
+import LevelBackground from './components/LevelBackground';
 import logo from './images/logo.png';
 
-class App extends Component {
-  config = {
-    itemTypes: {
-      // type: spawn rate (weighting)
-      litter: 20,
-      rock: 5,
-      bush: 5,
-      flower: 5,
-      mushroom: 5,
-    },
-    spawnRate: 1.2, // Hz
-    spawnRateRnd: 1.79, // randomization factor
-    spawnHeight: 100, // height of item spawn area in pixels
-    spawnFloor: 0, // offset from bottom of game "level" in pixels
-    itemLifetime: 10 * 1000, // 10 seconds (should be longer than CSS animation time)
-  }
 
-  constructor() {
-    super();
+// Configuration Information
+const config = {
+  itemTypes: {
+    // type: spawn rate (weighting)
+    litter: 20,
+    rock: 5,
+    bush: 5,
+    flower: 5,
+    mushroom: 5,
+  },
+  spawnRate: 1.2, // Hz
+  spawnRateRnd: 1.79, // randomization factor
+  spawnHeight: 150, // height of item spawn area in pixels
+  spawnFloor: 0, // offset from bottom of game "level" in pixels
+  itemLifetime: 8 * 1000, // 8 seconds (should be longer than CSS animation time)
+};
+
+// The time an item was last spawned
+let lastSpawn = null;
```

Comment:
Is it possible to implement this with `useState` as well?

Reply:
Ok migrated it to use state.  I had just left it the way it had been as a class component.